pkg/autodetect/main.go

```diff
@@ -197,6 +197,7 @@ func (b *Background) detectKafka(ctx context.Context, apiList *metav1.APIGroupLi
 
 func (b *Background) detectClusterRoles(ctx context.Context) {
 	tr := &authenticationapi.TokenReview{
+		ObjectMeta: metav1.ObjectMeta{Name: "jaeger-operator-TEST"},
```

Comment:
leftover?

Reply:
No, this is actually required now, otherwise it fails to validate ("token review has to have a name" or something...)

Reply:
Could this cause any issues when with multiple operands in the same namespace or multiple operators in the cluster?